net/advanced-operations/working-with-javascript/_index.md

```diff
@@ -106,6 +106,75 @@ A new property named JavaScript is added in Document class which has JavaScript
 
 {{< gist "aspose-pdf" "7e1330795d76012fcb04248bb81d45b3" "Examples-CSharp-AsposePDF-Working-Document-AddRemoveJavascriptToDoc-AddRemoveJavascriptToDoc.cs" >}}
 
+### Setting Expiry Date of a PDF Document Using JavaScript Actions
+
+Aspose.PDF allows you to set an expiry date for a PDF document by embedding JavaScript Actions. This functionality ensures that the PDF becomes inaccessible after a specified date and time, enhancing document security and control. By leveraging JavaScript Actions, you can define precise expiration conditions down to the second, ensuring that the document's accessibility is tightly regulated.
```

Comment:
Aspose.PDF allows you to set an expiry date for a PDF document by embedding JavaScript Actions. This functionality ensures the PDF becomes inaccessible after a specified date and time, enhancing document security and control. By leveraging JavaScript Actions, you can define precise expiration conditions down to the second, ensuring the document's accessibility is tightly regulated.